frontend/src/lib/components/PropertyGroupFilters/PropertyGroupFilters.tsx

```diff
@@ -0,0 +1,180 @@
+                                                style={{ fontSize: 16, color: 'var(--primary-alt)' }}
+                                            />
+                                        </Row>
+                                        <PropertyFilters
+                                            orFiltering={true}
+                                            propertyFilters={group.groups}
+                                            onChange={(properties) => {
+                                                setPropertyFilters(properties, propertyGroupIndex)
+                                            }}
+                                            pageKey={`trends-filters-${propertyGroupIndex}`}
+                                            taxonomicGroupTypes={taxonomicGroupTypes}
+                                            eventNames={eventNames}
+                                            propertyGroupType={group.type}
+                                        />
+                                    </div>
+                                    {propertyGroupIndex !== filtersWithNew?.groups.length - 1 && (
+                                        <Row className="text-small primary-alt">
+                                            <b>{filtersWithNew.type}</b>
+                                        </Row>
+                                    )}
+                                </>
+                            )
+                        })}
+                    </BindLogic>
+                </div>
+            )}
+            <div>
+                <Button
+                    style={{
+                        color: 'var(--primary)',
+                        border: 'none',
+                        boxShadow: 'none',
+                    }}
+                    className="mb"
+                    icon={<PlusOutlined />}
+                    onClick={() => addFilterGroup()}
+                >
+                    Add filter group
+                </Button>
+            </div>
+        </>
+    )
+}
+
+export enum AndOr {
+    AND = 'AND',
+    OR = 'OR',
+}
+
+interface AndOrFilterSelectProps {
+    onChange: (type: AndOr) => void
+    value: string
+    topLevelFilter?: boolean
+}
+
+export function AndOrFilterSelect({ onChange, value, topLevelFilter }: AndOrFilterSelectProps): JSX.Element {
+    return (
+        <Row align="middle" className="and-or-filter">
+            <span className="ml-05">Match</span>
+            <Select
+                optionLabelProp="label"
+                dropdownClassName="and-or-filter-select"
+                style={{ marginLeft: 8, marginRight: 8 }}
+                defaultValue={AndOr.AND}
+                onChange={(type) => onChange(type)}
+                dropdownMatchSelectWidth={false}
+                placement={topLevelFilter ? 'bottomRight' : 'bottomLeft'}
+            >
+                <Select.Option value={AndOr.AND} label="all" className="condition-option">
```

Comment:
I don't think this particularly hurts for it to be bolded right now, especially since we're not even sure if we want to keep this description as indicated by an above comment